Tests/Unit/System/Configuration/ExtensionConfigurationTest.php

```diff
@@ -74,8 +74,8 @@ public function testIsGetIsUseConfigurationMonitorTablesConfigured()
             ['useConfigurationMonitorTables' => 'pages, tt_content']
         );
         $expectedResult = [
-            'pages' => true,
-            'tt_content' => true
+            'pages' => 0,
```

Comment:
Is this really a wanted change? Becuase pages should also be 1 from my thinking and also we change the values from boolean to int?